examples/pip_requirements/pip_requirements.py

```diff
@@ -47,24 +47,22 @@ def main():
         run_id = run.info.run_id
 
         # Get the expected mlflow version
-        mlflow_version = Version(mlflow.__version__)
-        mlflow_version_range = (
-            f"mlflow<{mlflow_version.major + 1},>={mlflow_version.major}.{mlflow_version.minor}"
-        )
+        mlflow_version_raw = Version(mlflow.__version__)
```

Comment:
This is just to align the examples test to the new explicit version pinning from #8422

Reply:
Can we file this as a separate PR? When customers ask about the behavior change, I'd like to be able to point them to a standalone PR (burying significant changes like this in an example might not be thrilling for users). We should also have a release note just for this.